### DISCUSSION TITLE
Covalent start hangs on SUSE Linux Enterprise Server 15, Python 3.8

### DISCUSSION THREAD

**Discussion by pradeepmantha** [AgnostiqHQ/covalent#1894] [General]
I am using HPC machine with SLURM as resoruce manager.

After allocating a node, it seems covalent start hangs for ever, and coudlnt' even kill it using kill -9 command

```uname -a
Linux login06 5.14.21-150400.24.81_12.0.87-cray_shasta_c #1 SMP Mon Nov 6 21:51:33 UTC 2023 (e30c7c1) x86_64 x86_64 x86_64 GNU/Linux
~> lsb_release -a
LSB Version:	n/a
Distributor ID:	SUSE
Description:	SUSE Linux Enterprise Server 15 SP4
Release:	15.4
Codename:	n/a
```

``` > covalent start




^C^C^C^C^C^Z^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C```

Any covalent commands stuck , not sure how to go debugging about this.

**Comment by santoshkumarradha** [comment]
Hi @pradeepmantha,

This most definitely smells like bad installation or config corruption.

Can you try if `covalent logs` works? If so can you paste the output of that here ? 

if that fails, can you try manually deleting the configuration files and then uninstalling and reinstalling covalent ? You should find them here - 

```bash
rm -rf ~./config/covalent
rm -rf ~./cache/covalent
```

Also all of covalent is completely pythonic, so ideally this should  nuke everything `pkill -9 python`

**Comment by santoshkumarradha** [comment]
Noting down here might be related to https://github.com/AgnostiqHQ/covalent/discussions/1891

**Reply by pradeepmantha** [reply]
Thanks it's solved now. It seems on the HPC machine the default config file creation location need to be changed. After changing it, the server started.
